.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+### Describe Your Changes
+
+Please provide a brief description of the changes you made. Be as specific as possible to help others understand the purpose and impact of your modifications.
+
+### Checklist (Optional for External Contributions)
+
+- [ ] Include a link to the GitHub issue in the PR description, if possible.
+- [ ] Add an item to the [Changelog](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/docs/CHANGELOG.md) that describes the problem you're addressing and how you're solving it.
+
+  Tips for writing a good changelog message::
+
+    * Write a human-readable changelog message that describes the problem and solution.
+    * Include a link to the issue or pull request in your changelog message.
+    * Use specific language identifying the fix, such as an error message, metric name, or flag name.
+    * Provide a link to the relevant documentation for any new features you add or modify.
```

Comment:
```suggestion
```

I don't think someone needs to read it on each PR.

Reply:
this message has several goals 
- provide information for every new engineer
- remind everyone who create PRs to do some steps

so i don't think this repeatable PR to PR information is useless